awsconfigs/apps/jupyter-web-app/configs/spawner_ui_config.yaml

```diff
@@ -17,10 +17,10 @@
 spawnerFormDefaults:
   image:
     # The container Image for the user's Jupyter Notebook
-    value: public.ecr.aws/j1r0q0g6/notebooks/notebook-servers/jupyter-scipy:v1.5.0
+    value: kubeflownotebookswg/jupyter-scipy:v1.6.0
```

Comment:
lets change the default to DLC image `public.ecr.aws/c9e4w0g3/notebook-servers/jupyter-tensorflow:2.6.3-cpu-py38-ubuntu20.04-v1.8`

Reply:
changed.